src/transformers/tokenization_utils_base.py

```diff
@@ -3269,6 +3278,9 @@ def pad(
                 - `'np'`: Return Numpy `np.ndarray` objects.
             verbose (`bool`, *optional*, defaults to `True`):
                 Whether or not to print more information and warnings.
+            split_special_tokens ( `bool`, *optional*, default to `False`):
+                Whether or not the special tokens should be encoded. If `True`, they are encoded, and will
+                be split by the tokenizer. This should be activated for safe prompting.
```

Comment:
Update with a mention saying that passing it to the forward call will affect the internal state

Reply:
Also bit weird that the doc for this is int he pad function, there might be a better place as pad does not do anything about it!